[Documentation] Nvidia style guide edits to pipeline visualization (#1328)
nvidia-holoscan/holohub: applications/pipeline_visualization/README.md

### Application (`cpp/` and `python/`)

> The C++ and Python applications demonstrates a basic Holoscan pipeline with data logging:

Author: changing locally

### Python Visualizers (`visualizer/`)

There are two Python visualizers. One is [static](#static-visualizer-visualizer_staticpy) and one is [dynamic](#dynamic-visualizer-visualizer_dynamicpy) and they both display:

- Real-time line plots of tensor data
- Stream name (operator.port format)
- IO type (Input/Output)
- Acquisition timestamp (nanoseconds)
- Publish timestamp (nanoseconds)

> Use `start_visualizer.sh` to set the required Python path to the [flatbuffers](#flatbuffers-schemas-schemas) definitions and starts the visualizer. The script takes one parameter, its values are:

Author: changing locally

- `dynamic`, to start the dynamic visualizer.
- `static`, to start the static visualizer.

#### Static Visualizer (`visualizer_static.py`)

The static visualizer can be used when the output data and format of the Holoscan pipeline is known or some data needs special formatting.

- Displays predefined data streams: `source.out`, `modulate.in`, `modulate.out`, `sink.in`
- Best for applications with known, fixed operator topology
- All graphs are created upfront and updated as data arrives

#### Dynamic Visualizer (`visualizer_dynamic.py`)

- Automatically discovers and creates graphs for new data streams
- Ideal for applications with dynamic or unknown operator configurations
- Graphs are created on-the-fly as new unique IDs are detected

> ### FlatBuffers Schemas (`schemas/`)

Author: went to my local copy to move some sections around.  Also moved Troubleshooting to be the very last section.  I still have reservations about how the rest of the topics are wedged in this readme.  The H2 level has a chaotic conversation feel. 

Also still unsure how the FlatBuffers work.  I will likely do some web searching....to see if I can glean anything.

- **message.fbs**: Top-level message structure with metadata
- **tensor.fbs**: Tensor data structure based on DLPack

## Visualizer Python Prerequisites

All dependencies to run the application are installed automatically when using the `holohub run` command inside

> the Holohub container. Because the visualizer is run outside the Holohub container, its [dependencies](#visualizer-python-dependencies) must be

Reviewer: The application has been updated to install these dependencies automatically inside the container but the README was not reflecting that.

```suggestion
the Holohub container. However, if you are running the visualizer outside the Holohub container, its [dependencies](#visualizer-python-dependencies) must be
```

- **message.fbs**: Top-level message structure with metadata
- **tensor.fbs**: Tensor data structure based on DLPack

> ## Visualizer Python Prerequisites

Author: I took a shot at what these prereq's were for. **Please verify.**

**Solution**:

1. Ensure the FlatBuffers files were generated during build.
1. Set `PYTHONPATH` correctly:

    ```bash
    export PYTHONPATH=$PYTHONPATH:/path/to/build/applications/pipeline_visualization/flatbuffers/
    ```

> 1. Verify that the files exist in the build directory.

Reviewer: ```suggestion
1. Ensure the FlatBuffers files were generated during build.
2. Set `PYTHONPATH` correctly:

    ```bash
    export PYTHONPATH=$PYTHONPATH:/path/to/build/applications/pipeline_visualization/flatbuffers/
    ```

3. Verify that the files exist in the build directory.
```
I know it renders correctly but some people just read the markdown instead of its rendering.
